taglibs/async/await-tag.js

```diff
@@ -23,6 +23,10 @@ function promiseToCallback(promise, callback, thisObj) {
             finalPromise = finalPromise.fail(function(err) {
                 callback(err);
             });
+        } else {
+            finalPromise = finalPromise.then(void 0, function(err) {
```

Comment:
Do you mind switching from `void 0` to `undefined`? If `undefined` !== `void 0` then we would be in a huge mess